firefox-ios/Client.xcodeproj/project.xcworkspace/xcshareddata/swiftpm/Package.resolved

```diff
@@ -95,8 +95,8 @@
       "kind" : "remoteSourceControl",
       "location" : "https://github.com/mozilla/rust-components-swift.git",
       "state" : {
-        "revision" : "003a732adbaeb9c009a7f0f2a91832a584a143a0",
-        "version" : "132.0.20240831050348"
+        "revision" : "1601e925c75912e9f891c31c2b62b102b90fb274",
```

Comment:
I don't think we should update the packages in feature/ bug PRs. Usually we have automated PRs that will update them. Do you mind reverting this change?